### PR TITLE
Feature/guildcommand

### PR DESCRIPTION
**Pull request by JamesHawkinss**
# Pull request

 - [ ] I have read and agreed to the [CODE OF CONDUCT](https://github.com/CascadeBot/CascadeBot/blob/master/CODE_OF_CONDUCT.md).
 - [ ] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [trello](https://trello.com/b/gUQehy4l/bot).
 - [ ] I have tested all of my changes.
 - [ ] I have followed the style of the rest of the project.
 
## Added/Changed
What did you add and/or change for this.

## Feature
If this is a new feature, why should we add it to the bot.

### CODE REVIEW

**Comment by JamesHawkinss** [comment]
[Guild command](https://trello.com/c/gxTuwQ6w/21-guild-command)

File: src/main/java/com/cascadebot/cascadebot/commands/developer/GuildCommand.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import com.cascadebot.cascadebot.commandmeta.Module;
+import com.cascadebot.shared.SecurityLevel;
+import net.dv8tion.jda.core.entities.Member;
+
+import java.util.Set;
+
+public class GuildCommand implements ICommandRestricted {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/commands/developer/GuildCommand.java]: We should reply with **something** here. maybe for now just reply with `Do [prefix]guild save to save the guild data`
@binaryoverload at some point we should add automatic formatting to like context.reply()

File: src/main/java/com/cascadebot/cascadebot/commands/developer/GuildSaveSubCommand.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import com.cascadebot.cascadebot.commandmeta.ICommandExecutable;
+import com.cascadebot.cascadebot.data.mapping.GuildDataMapper;
+import com.cascadebot.cascadebot.permissions.CascadePermission;
+import net.dv8tion.jda.core.entities.Member;
+
+public class GuildSaveSubCommand implements ICommandExecutable {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        GuildDataMapper.getGuilds().invalidate(context.getGuild().getIdLong());
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/commands/developer/GuildSaveSubCommand.java]: This should probably be default only save the current guild, but accept guild id and `all`

File: src/main/java/com/cascadebot/cascadebot/commands/developer/GuildCommand.java (hunk 1 above)
Comment (binaryoverload) [reply src/main/java/com/cascadebot/cascadebot/commands/developer/GuildCommand.java]: What do you mean? @weeryan17

File: src/main/java/com/cascadebot/cascadebot/commands/developer/GuildSaveSubCommand.java (hunk 2 above)
Comment (binaryoverload) [reply src/main/java/com/cascadebot/cascadebot/commands/developer/GuildSaveSubCommand.java]: +1 for this

**Pull request merged**